gormschema/dialect/spanner/go.mod

```diff
@@ -0,0 +1,85 @@
+module ariga.io/atlas-provider-gorm/gormschema/dialect/spanner
```

Comment:
@giautm / @luantranminh, I prefer to avoid managing (and tagging) submodules.

Also, keep in mind the provider should work on the standalone mode too (which is the default for most users).